dbms/src/Storages/DeltaMerge/ScanContext.h

```diff
@@ -0,0 +1,85 @@
+// Copyright 2022 PingCAP, Ltd.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#pragma once
+
+#include <common/types.h>
+#include <fmt/format.h>
+#include <sys/types.h>
+#include <tipb/executor.pb.h>
+
+#include <atomic>
+
+namespace DB::DM
+{
+class ScanContext
```

Comment:
Prefer to add some doc comments so that others can easily understand the functionality of this class.